app/components/newsletter/newsletter-header-overview.hbs

```diff
@@ -118,14 +106,15 @@
     @isLoading={{this.isLoading}}
     @verifyButtonText={{t "send-campaign"}}
     @message={{t "send-campaign-warning"}}
-    @cancel={{action "toggleIsVerifying"}}
-    @verify={{action "sendCampaign"}} />
+    @cancel={{on "click" this.toggleIsVerifying}}
+    @verify={{on "click" this.sendCampaign}}
+  />
 {{/if}}
 {{#if (or this.testCampaignIsLoading this.newsletterHTML)}}
   <WebComponents::VlModal
     @isOverlay={{true}}
     @title={{t "send-test-campaign"}}
-    @closeModal={{action "clearNewsletterHTML"}}
+    @closeModal={{on "click" this.clearNewsletterHTML}}
```

Comment:
Dit is geen `on "click"`, hier geven we gewoon een functie door

Niet alle `{{action }}` mag je zomaar vervangen met `{{on "click"}}`
Goede regel: als je iets aan het doorgeven bent met `@` is het meestal geen on click.
On click is enkel voor buttons en menu items ofzo